MediaRecorder.html

```diff
@@ -254,11 +273,12 @@
                   following steps:
                     <ol>
                       <li>set <code>recording</code> to "inactive".</li>
-                      <li>Fire a <code>dataavailable</code> event containing
-                      <var>blob</var>.</li>
-                      <li>Fire a <code>stop</code> event.</li>
+                      <li>Fire an event named <code>dataavailable</code>
```

Comment:
Based on https://dom.spec.whatwg.org/#firing-events-example I think this can be "Fire an event named `dataavailable` at *target* using `BlobEvent` with its `blob` event intialized to *blob*" where *target* and *blob* are variables to be defined in previous steps.

Note that the `ErrorEvent` cases can also be rewritten on that form, although as long as it's unambiguous it doesn't matter if there's a "fire an error event" concept.